cirq-core/cirq/devices/superconducting_qubits_noise_properties.py

```diff
@@ -0,0 +1,269 @@
+# Copyright 2021 The Cirq Developers
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from dataclasses import dataclass, field
+from typing import Dict, TYPE_CHECKING, List, Set
+
+from cirq import ops, protocols, devices
+from cirq.devices.noise_utils import (
+    OpIdentifier,
+    decoherence_pauli_error,
+)
+
+if TYPE_CHECKING:
+    import cirq
+
+SINGLE_QUBIT_GATES: Set[type] = {
+    ops.ZPowGate,
+    ops.PhasedXZGate,
+    ops.MeasurementGate,
+    ops.ResetChannel,
+}
+SYMMETRIC_TWO_QUBIT_GATES: Set[type] = {
+    ops.FSimGate,
+    ops.PhasedFSimGate,
+    ops.ISwapPowGate,
+    ops.CZPowGate,
+}
+ASYMMETRIC_TWO_QUBIT_GATES: Set[type] = set()
+TWO_QUBIT_GATES = SYMMETRIC_TWO_QUBIT_GATES | ASYMMETRIC_TWO_QUBIT_GATES
+
+
+# TODO: missing per-device defaults
+@dataclass
+class SuperconductingQubitsNoiseProperties(devices.NoiseProperties):
```

Comment:
nit: If `SuperconductingQubit` is used as an adjective to describe `NoiseProperties`, it should probably be singular and not plural.

Reply:
My reasoning for this was that the properties describe the entire device, not just a single qubit, and a plural helps to avoid that misconception.